src/registrar/templates/emails/domain_request_withdrawn.txt

```diff
@@ -0,0 +1,26 @@
+{% autoescape off %}{# In a text file, we don't want to have HTML entities escaped #}
+Hi {{ application.submitter.first_name }}.
+
+Your .gov domain request has been withdrawn.
+DOMAIN REQUESTED: {{ application.requested_domain.name }}
+REQUEST #: {{ application.id }}
+STATUS: Withdrawn
+
+
+YOU CAN EDIT YOUR WITHDRAWN REQUEST
+
+The details of your withdrawn request are included below. You can edit and resubmit this application by logging into the registrar. <https://manage.get.gov/>.
+
+THANK YOU
+
+.Gov helps the public identify official, trusted information. Thank you for requesting a .gov domain.
+
+----------------------------------------------------------------
+
```

Comment:
There seems to be an extra whitespace here (lines 18-19), did the template want this included?

Reply:
Good eye for detail! I did follow the spacing pattern in the `src/registrar/templates/emails/status_change_action_needed.txt` file so I think it should be fine though 😅

Reply:
Ah, if its the template then its probably okay!